packages/teams-js/src/public/mail.ts

```diff
@@ -14,6 +14,10 @@ export namespace mail {
         throw 'Not Supported';
       }
 
+      if (!openMailItemParams.itemId || !openMailItemParams.itemId.trim()) {
+        throw new Error('Must supply an item id to openMailItem');
```

Comment:
This message doesn't match the test expectation and is failing the tests (yay!)

Reply:
I am excited to announce that this is because I forgot to save the other half of the change